pages/api/emotionEntries/[id].js

```diff
@@ -0,0 +1,21 @@
+import EmotionEntry from "@/db/models/emotionEntry";
+import dbConnect from "../../../db/connect";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const { id } = request.query;
+
+  if (request.method === "GET")
+    try {
+      const emotionEntry = await EmotionEntry.findById(id).populate("type");
+
+      if (!emotionEntry) {
+        return response.status(404).json({ status: "Not Found" });
```

Comment:
it would be a bit cleaner to separate this into two lines. One for sending the response, one for stopping the "handler" function by calling "return".
```suggestion
        response.status(404).json({ status: "Not Found" });
        return 
```